pymagicc/api.py

```diff
@@ -66,11 +66,16 @@ def __enter__(self):
     def __exit__(self, *args, **kwargs):
         self.remove_temp_copy()
 
+    def get_executable(self):
+        raise NotImplementedError
+
     def create_copy(self):
         """
         Initialises a temporary directory structure and copy of MAGICC
         configuration files and binary.
         """
+        if self.executable is None or not isfile(self.executable):
+            raise ValueError('Could not find MAGICC{} executable: {}'.format(self.version, self.executable))
```

Comment:
How about FileNotFoundError

https://docs.python.org/3/library/exceptions.html#FileNotFoundError

Reply:
Now that we're Python3 only we can use Python3 exceptions :-)